src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Logs.component.tsx

```diff
@@ -384,84 +382,108 @@ function LogsComponent({
+                                                        ...base,
+                                                        fontWeight: 600,
+                                                        fontSize: '10px',
+                                                        color: 'var(--n-700)',
+                                                        marginLeft: 0,
+                                                    }),
+                                                    singleValue: (base, state) => ({
+                                                        ...base,
+                                                        fontWeight: 600,
+                                                        color: '#000A14',
+                                                        direction: 'rtl',
+                                                        textAlign: 'left',
+                                                        marginLeft: '2px',
+                                                    }),
+                                                    indicatorsContainer: (provided, state) => ({
+                                                        ...provided,
+                                                    }),
+                                                    dropdownIndicator: (base, state) => ({
+                                                        ...base,
+                                                        padding: '0',
+                                                    }),
+                                                }}
+                                                components={{
+                                                    IndicatorSeparator: null,
+                                                    Option: (props) => (
+                                                        <Option
+                                                            {...props}
+                                                            showTippy={true}
+                                                            style={{ direction: 'rtl' }}
+                                                        />
+                                                    ),
+                                                }}
+                                            />
+                                        </div>
                                     </div>
-                                </div>
-                            </React.Fragment>
+                                </React.Fragment>
+                            </>
                         )}
 
                         {(podContainerOptions?.containerOptions ?? []).length > 0 && (
                             <React.Fragment>
+                                <div className="h-16 dc__border-right ml-8 mr-8"></div>
                                 <div className="cn-6 ml-8 mr-10">Container </div>
-                                <div style={{ width: '150px' }}>
```

Comment:
is removal of this intentional?

Reply:
Yes, there was demand for content driven select's width

Reply:
Added max width of 200px as decided